public/locales/en/common.json

```diff
@@ -146,5 +146,21 @@
   "taxIdentificationNumber": "Tax Identification Number",
   "tinRequired": "Tax Identification Number is Required",
   "tinText": "TIN:",
-  "returnTo": "Return to"
-}
\ No newline at end of file
+  "returnTo": "Return to",
+  "donateWithPlanetCash": "Donate with PlanetCash",
+  "donationOnBehalfText": "This donation is on behalf of someone",
+  "donationOnBehalfOf": "This donation is on behalf of {{firstName}} {{lastName}}",
+  "usePlanetCash": "Use PlanetCash for this donation",
+  "balance": "Balance",
+  "credit": "Credit",
+  "addBalance": "Add Balance",
+  "onBehalfOf": "On behalf of",
+  "inValidField": "{{fieldName}} is invalid",
+  "onBehalfNotification": "On Behalf Notification",
+  "errorStatus500": "Something went wrong please try again soon!",
+  "errorStatus503": "App is undergoing maintenance, please check status.plant-for-the-planet.org for details",
+  "pCashTopUpSuccess": "Your {{frequency}}PlanetCash donation of {{amount}} was successful.",
```

Comment:
"pCashTopUpSuccess"  is redundant to. repeat donations thank you message. this can be made generic.

Your {{frequency}} {{purpose}} donation of {{amount}} was successful. can be generic for all donations.

Reply:
Done